cognite/neat/_graph/extractors/_classic_cdf/_classic.py

```diff
@@ -372,6 +384,36 @@ def _extract_data_sets(self):
             )
             yield from DataSetExtractor(data_set_iterator, **self._extractor_args).extract()
 
+    def _extract_asset_parent_data_sets(self):
+        if self._asset_parent_uri_by_id:
+            for chunk in self._chunk(
+                list(self._asset_parent_uri_by_id.keys()), description="Extracting asset parent data sets"
+            ):
+                assets = self._client.assets.retrieve_multiple(id=list(chunk), ignore_unknown_ids=True)
```

Comment:
We include the dataset such that we can set space consistently based on it, no matter which hierarchy we use as the source for the extraction.